pygmt/tests/test_meca.py

```diff
@@ -54,7 +118,7 @@ def test_meca_spec_dict_list():
 
 
 @pytest.mark.mpl_image_compare
-def test_meca_spec_dataframe():
+def test_meca_spec_two_dataframe():
     """
     Test supplying a pandas.DataFrame containing focal mechanisms and locations
     to the spec parameter.
```

Comment:
```suggestion
    Test supplying two focal mechanisms and their locations by passing a pandas.DataFrame 
    to the spec parameter.
```

I think this makes it more clear why "two" is in the function name.